utilities/log_expvars.py

```diff
@@ -0,0 +1,71 @@
+import time
+import datetime
+import requests
+
+from provisioning_config_parser import hosts_for_tag
+
+
+def log_expvars():
+    usage = """
+    usage: log_expvars.py"
+    """
+
+    # Get gateload ips from ansible inventory
+    lgs_host_vars = hosts_for_tag("load_generators")
+    lgs = [lg["ansible_ssh_host"] for lg in lgs_host_vars]
+
+    print("Monitoring gateloads: {}".format(lgs))
+    lgs_with_port = [lg + ":9876/debug/vars" for lg in lgs]
+
+    # Get sync_gateway ips from ansible inventory
+    sgs_host_vars = hosts_for_tag("sync_gateways")
+    sgs = [sgv["ansible_ssh_host"] for sgv in sgs_host_vars]
+
+    print("Monitoring sync_gateways: {}".format(sgs))
+    sgs_with_port = [sg + ":4985/_expvar" for sg in sgs]
+
+    endpoints = list()
+    endpoints.extend(lgs_with_port)
+
+    target_test_filename = "perf_test.log"
+
+    with open(target_test_filename, "w") as f:
+
+        # in seconds
+        test_time = 0
+        collect_interval = 120
+
+        while test_time < 1200:
+
+            for endpoint in endpoints:
+                try:
```

Comment:
Can you refactor the body of this loop into `log_expvars(endpoint)`?  It will make the code easier to read.